xsuite/src/world/chainSimulator.ts

```diff
@@ -0,0 +1,62 @@
+import { spawn } from 'node:child_process';
+
+export const startChainSimulator = (
+  port: number = 8085,

```

Comment:
Unfortunately this does not seem possible, so tests with the Chain Simulator should not be run in parallel (eg with Vitest: `vitest run --no-file-parallelism`)

This is because the Chain Simulator doesn't return back the port it uses.
And it also doesn't fail to start if the same port is used to spawn multiple processes, just tried this and I could spawn two chain simulator processes successfully with port 8085.

If parallel execution is wanted, the end user can specify another port.

Reply:
The chain simulator must be changed then.

Reply:
Done